### PR TITLE
feat: bump Connectors to 0.2.0

### PR DESCRIPTION
**Pull request by tmetzke**
⚠️ Only merge when Connectors 0.2.0 is available.

### CODE REVIEW

**Review by berndruecker — APPROVED** [review]

**Comment by menski** [comment]
Just to make sure the next version will be 0.2.0 of the bundle as it described in this PR. Asking because I just saw a [0.3.0-alpha5](https://hub.docker.com/layers/camunda/connectors-bundle/0.3.0-alpha5/images/sha256-24697c54194bb04af17b42f6929bfd5e85dfdc38da26e6f89131e752c40e5e69?context=explore) on docker hub.

**Comment by berndruecker** [comment]
I think it is correct - but I actually had the same first reaction :-) I wonder if it make sense to align the bundle version with the latest connector-sdk version @tmetzke? 
I know that in theory they are independent, but I think in practice they are not, at least until we reach 1.0.
Feel free to ignore though :-)

**Comment by tmetzke** [comment]
So, the alpha release was an accident, actually. I couldn't release the runtime and bundle independently at that time but realized it just too late.

I think we could align their versions for good, yes. Should make it easier to follow along with the SDK releases for now. I'll adjust the PR here accordingly, thanks for bringing it up 👍

**Comment by menski** [comment]
Just for the record we [actively decided](https://github.com/camunda/team-connectors/issues/142) to disconnect the versions of the sdk and the bundle, to allow us to update connectors, bug fixes etc without depending on a new SDK release, assuming the SDK is quite stable. Similar how we do it for the helm charts, and spring-zeebe for example. I didn't wanted to push for aligning the versions, just noticed the different image version on docker hub.

**Comment by tmetzke** [comment]
Thanks, @menski, I was actually not aware that we had this discussion 👍 
Let's keep it separate then, I agree with the reasoning with decided on previously.

Since I enabled independent releases of Docker images in the `connectors-bundle` repository, this should be easy to achieve. I will test this today.

@menski, can you delete `0.3.0-alpha5` of `connectors-bundle` in the Docker hub?

**Comment by menski** [comment]
> @menski, can you delete 0.3.0-alpha5 of connectors-bundle in the Docker hub?

Done, but I noticed that the `latest` tag also points to the `0.3.0-alpha5` image. If you tell me which is the latest image version, I could update the `latest` tag.

**Comment by tmetzke** [comment]
> > @menski, can you delete 0.3.0-alpha5 of connectors-bundle in the Docker hub?
> 
> Done, but I noticed that the `latest` tag also points to the `0.3.0-alpha5` image. If you tell me which is the latest image version, I could update the `latest` tag.

Ah, yeah, sorry for that. The latest one is still [0.1.2](https://hub.docker.com/layers/camunda/connectors-bundle/0.1.2/images/sha256-103a76ae6ebd4459529198d84fde86477cf1db072815ba6f86e62f6a3dfc2407?context=explore)

**Comment by menski** [comment]
I pushed `0.1.2` as `latest`

**Comment by berndruecker** [comment]
Don't want to make too much fuzz here now - so feel free to ignore - but from a user perspective I find it way easier if at least the 0.3 is aligned (so 0.3.x for the bundle is based on 0.3.x of the SDK). You can still differ in the patch version. That's what I do with spring-zeebe. 

I think we can completely disconnect if the SDK is stable - but at the moment it is pretty confusing that I use a bundle 0.2, because I want to base on the latest connector sdk 0.3 (even internally - what about externally?).

**Comment by berndruecker** [comment]
But hey - probably don't touch the already made decision now to just not open up another discussion pot :-) Will be outdated with 1.0 anyway - so let's focus to get there!

File: docker-compose-core.yaml [hunk 1]
```diff
@@ -64,11 +64,11 @@ services:
       - elasticsearch
 
   connectors: # https://docs.camunda.io/docs/components/integration-framework/connectors/out-of-the-box-connectors/available-connectors-overview/
-    image: camunda/connectors-bundle:${CAMUNDA_CONNECTORS_VERSION:-0.1.2}
+    image: camunda/connectors-bundle:${CAMUNDA_CONNECTORS_VERSION:-0.2.0}
```
Comment (tmetzke) [inline docker-compose-core.yaml:67]:
```suggestion
    image: camunda/connectors-bundle:${CAMUNDA_CONNECTORS_VERSION:-0.2.1}
```

File: README.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ per line. The secrets will then be available in the Connector runtime with the
 format `secrets.NAME`.
 
 To add custom Connectors either create a new docker image bundling them as
-described [here](https://github.com/camunda/connector-sdk/tree/main/runtime#docker-job-worker-runtime-image).
+described [here](https://github.com/camunda-community-hub/spring-zeebe/tree/master/connector-runtime#via-docker).
```
Comment (tmetzke) [inline README.md:81]:
```suggestion
described [here](https://github.com/camunda/connectors-bundle/tree/main/runtime).
```

**Pull request merged**